README.md

```diff
@@ -190,5 +195,55 @@ As we add more features to CDP that require additional processing or resources w
 continue to try to minimize their costs wherever possible. Further, if a feature is
 optional, we will create a flag that maintainers can set to include or exclude the
 additional processing or resource usage.
+See [Upgrades and New Features](#upgrades-and-new-features) for more information.
+
+## Upgrades and New Features
+
+Answering the question:
+_"How does my instance of CDP receive updates and new features?"_
+
+In general, all updates and upgrades are handled easily for you through
+automated systems that run using GitHub Actions on your repository.
+
+Upgrades are delivered either just before processing time (during package installation)
+or as a part of weekly system checks and auto-deployments.
+
+#### Backend Pipeline Upgrades
+
+Due to how we have constructed how event processing works, every time CDP
+pipelines are initiated, they also install the latest non-breaking-changes version of
+the [cdp-backend](https://github.com/CouncilDataProject/cdp-backend) package.
+
+In this way, every time your pipeline runs to gather events, index events, or
+index pieces of legislation, you can be sure you are up-to-date.
+
+#### Backend Infrastructure Upgrades
+
+Every week, your repository will automatically check the status of your infrastructure
+and deploy updates if needed.
+
+#### Frontend Web Application Upgrades
+
+Every week, your repository will automatically build and deploy the CDP web application,
+during the build process any non-breaking changes and new features will automatically be
+pulled in.
+
+### Breaking Changes
+
+In the case we release a new version of
+[cdp-backend](https://github.com/CouncilDataProject/cdp-backend) or
+[cdp-frontend](https://github.com/CouncilDataProject/cdp-frontend) that includes
+breaking changes we will also release an upgrade guide that you will be able to find
+in this repository.
+
+In the case we can fully automate the upgrade, we will include a script to do so that
+CDP instance maintainers will simply have to run.
+
+### Notification of Updates and Release Notes
+
+We are still in the process of developing a notification system for alerting CDP
+instance maintainers that a new version of `cdp-backend` or `cdp-frontend` was
+released. These notifications will include details on new features, bugfixes, and
+if there is anything the instance maintainer needs to do to enable the release.
```

Comment:
Maybe add a link to become a watcher of the two repos so you can be notified of releases?

Reply:
This is a much simpler system and I appreciate this. :joy:

Reply:
🤔 I'm not sure if there is a direct link to watch something on GitHub actually

Reply:
I don't think there is either, but I think we can include instructions that say "Click watch, select custom, and select Release, then Apply." You will receive emails when that repository releases new versions,